hackerank/js-practice/birthday-cake-candles.js

```diff
@@ -1,32 +1,25 @@
-'use strict';
+"use strict";
```

Comment:
Why do you change `'` by `"`?

Reply:
Because of my Prettier configuration. If you want, we can discuss a common configuration